_layouts/startpage.html

```diff
@@ -0,0 +1 @@
+{{ content }}
```

Comment:
To display the start/landing page correctly an empty layout is needed which doesn't have DOCTYPE, html or any html tag in it, because the `index.html` itself comes with that.